eng/common/scripts/Create-APIReview.ps1

```diff
@@ -0,0 +1,87 @@
+[CmdletBinding()]
+Param (
+  [Parameter(Mandatory=$True)]
+  [string] $ArtifactPath,
+  [Parameter(Mandatory=$True)]
+  [string] $APIViewUri,
+  [Parameter(Mandatory=$True)]
+  [string] $APIKey,
+  [Parameter(Mandatory=$True)]
+  [string] $APILabel,
+  [string] $PackageName = ""
+)
+
+
+# Submit API review request and return status whether current revision is approved or pending or failed to create review
+function Submit-APIReview($packagename, $filePath, $uri, $apiKey, $apiLabel){
+    
+    $multipartContent = [System.Net.Http.MultipartFormDataContent]::new()
+    $FileStream = [System.IO.FileStream]::new($filePath, [System.IO.FileMode]::Open)
+    $fileHeader = [System.Net.Http.Headers.ContentDispositionHeaderValue]::new("form-data")
+    $fileHeader.Name = "file"
+    $fileHeader.FileName = $packagename
+    $fileContent = [System.Net.Http.StreamContent]::new($FileStream)
+    $fileContent.Headers.ContentDisposition = $fileHeader
+    $fileContent.Headers.ContentType = [System.Net.Http.Headers.MediaTypeHeaderValue]::Parse("application/octet-stream")
+    $multipartContent.Add($fileContent)
+
+
+    $stringHeader = [System.Net.Http.Headers.ContentDispositionHeaderValue]::new("form-data")
+    $stringHeader.Name = "label"
+    $StringContent = [System.Net.Http.StringContent]::new($apiLabel)
+    $StringContent.Headers.ContentDisposition = $stringHeader
+    $multipartContent.Add($stringContent)
+
+    $form = @{
+        'label' = $apiLabel;
+        'file' = Get-content -Path $filePath
```

Comment:
We might want to use `-Raw` for the Get-Content call.

Reply:
This object is not used. I've removed this code.